third_party/2and3/emoji/core.pyi

```diff
@@ -1,8 +1,10 @@
-from typing import Dict, List, Pattern, Text, Tuple, Union
+from typing import Dict, List, Optional, Pattern, Text, Tuple, Union
 
+_EMOJI_REGEXP: Optional[Pattern[Text]]
```

Comment:
Do you think it makes sense to define this here in the stubs? Is there a situation where a user would use this instead of `get_emoji_regexp()`?

Reply:
Only reason I defined this was that `_DEFAULT_DELIMITER` is already typed despite being private constant. I can remove it.